src/bioregistry/export/prefix_maps.py

```diff
@@ -34,6 +35,30 @@ def generate_context_json_ld():
             json.dump(fp=file, indent=4, sort_keys=True, obj=get_collection_jsonld(key))
 
 
+@click.command()
+def generate_shacl_prefixes():
```

Comment:
just put this inside the already existing main function, perhaps we need to rename it to be more general though

Reply:
Fixed in https://github.com/biopragmatics/bioregistry/pull/318/commits/6f8b0e0749b553c03c187fc01825ec997c646022